crates/apub/src/collections/community_featured.rs

```diff
@@ -76,7 +76,10 @@ impl ApubObject for ApubCommunityFeatured {
   {
     let mut posts = apub.ordered_items;
     if posts.len() as i64 > FETCH_LIMIT_MAX {
-      posts = posts[0..(FETCH_LIMIT_MAX as usize)].to_vec();
+      posts = posts
+        .get(0..(FETCH_LIMIT_MAX as usize))
+        .expect("limiting the community post count")
```

Comment:
I thought this was a false positive but it actually panicks. But use `unwrap_or_default()`.